### PR TITLE
Add awesome animated gif to Readme

### PR DESCRIPTION
**Pull request by farazfazli**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 Apex is a small tool for deploying and managing [AWS Lambda](https://aws.amazon.com/lambda/) functions. With shims for languages not yet supported by Lambda, you can use Golang out of the box.
 
+<p align="center">
+  <img src="https://github.com/farazfazli/apex/blob/master/assets/terminal.gif" alt="Sublime's custom image"/>
```
Comment (tj) [inline Readme.md]: src="assets/terminal.gif" :D, it's hard-coded to yours right now

Comment (tj) [inline Readme.md]: looks low res on retina, I can try one on my machine and see if it's less blurry

Comment (farazfazli) [inline Readme.md]: Hahaha, yes it is...we can change it after the merge! I will try recording it again.

**Comment by tj** [comment]
out

**Comment by tj** [comment]
hmm that looks blurry too wtf tj/stack's one doesn't look bad

**Comment by tj** [comment]
nvm retina ones are like 30mb

File: Readme.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 
 Apex is a small tool for deploying and managing [AWS Lambda](https://aws.amazon.com/lambda/) functions. With shims for languages not yet supported by Lambda, you can use Golang out of the box.
 
+<p align="center">
+  <img src="https://github.com/farazfazli/apex/blob/master/assets/terminal.gif" alt="Terminal Example"/>
```
Comment (turboMaCk) [inline Readme.md]: relative path will be much better I think.

**Comment by mthenw** [comment]
It's quite blurry on Retina, besides do we really need this?

**Comment by tj** [comment]
doesn't matter to me, I kinda like that the node one shows how fast deploys can be with lambda, but otherwise I think it probably works the way people would expect from the docs

**Comment by farazfazli** [comment]
I don't think you can center the image if you use a relative path with markdown.

**Comment by turboMaCk** [comment]
Why not? This does not makes sense. DOM itself is responsible for rendering. It does not not care about src location, just data.

**Comment by tj** [comment]
let's revisit if there's a way to make a non-blurry version that isn't multi-megabytes

**Comment by farazfazli** [comment]
@turboMaCk http://stackoverflow.com/a/12118349

**Comment by turboMaCk** [comment]
```html
<p align="center">
   <img src="assets/terminal.gif" alt="Terminal Example">
</p>
```

would work perfectly.

**Comment by farazfazli** [comment]
Oh okay, @turboMaCk I didn't know you could do that! Thanks! :smiley:

**Pull request closed without merging**